### ISSUE TITLE
Unsupported camera message → Weird font size

### ISSUE THREAD

**Issue by timur-davletshin** [darktable-org/darktable#9313]
1. Latest git dt snapshot
2. Open in dt any file of unsupported type
3. Error message appears (darktable could not load `some file', switching to lighttable now.)
4. Font size of that message at least twice bigger than all other on-screen messages. Should be the same size.

**Comment by Nilvus** [comment]
Could you share a screenshot? And an image to reproduce? Thanks

**Comment by timur-davletshin** [comment]
> Could you share a screenshot? And an image to reproduce? Thanks

https://raw.pixls.us/getfile.php/2058/nice/Hasselblad%20-%20Hasselblad%20X1D%20-%2016bit%20(4:3).3FR

**Comment by Nilvus** [comment]
> > Could you share a screenshot? And an image to reproduce? Thanks
> 
> https://raw.pixls.us/getfile.php/2058/nice/Hasselblad%20-%20Hasselblad%20X1D%20-%2016bit%20(4:3).3FR

And a screenshot with the message please

**Comment by timur-davletshin** [comment]
> And a screenshot with the message please

![big_msg](https://user-images.githubusercontent.com/65483426/123053973-17916500-d40d-11eb-8fdb-08cc7bd18f6c.png)

**Comment by Nilvus** [comment]
@timur-davletshin: your link doesn't work. Download won't start. I've tried to search it on raw.pixl.us but can't download it. Seems that I can't reproduce too this message by using CR3 files (that are not supported actually). Could you share the file by another way (a cloud link for example)?

**Comment by timur-davletshin** [comment]
@Nilvus https://drive.google.com/file/d/1cXqTUKkLh5pCuRuRm5PqSKCWRyd1f_s6/view?usp=sharing — try this one.

**Comment by Nilvus** [comment]
Forget last comment. Finally succeed after a long long time. Thanks.

**Comment by timur-davletshin** [comment]
@Nilvus looks like different unsupported file types trigger different messages. Hasselblad's file can be imported and even kinda processed if you rename them as TIFF (only builtin miniature will be processed).

**Comment by Nilvus** [comment]
I've checked and font size is about 50% higher. After testing it in same size as other messages, I found that wrong as this message error is particular and need to be check more. And font size is defined specifically to be different for this text. So it's not an issue but a wanted behavior. This is a particular message and I wouldn't set it same size as others for that reason. But maybe it's too big. It's set as 20 as other are set to 14. 16 seems better compromise.
